test/nbrowser/testServer.ts

```diff
@@ -67,6 +67,9 @@ export class TestServerMerged extends EventEmitter implements IMochaServer {
     this._starts++;
     const workerIdText = process.env.MOCHA_WORKER_ID || '0';
     if (reset) {
+      // Make sure this test server doesn't keep using the DB that's about to disappear.
+      this.closeDatabase();
```

Comment:
Needs an await here.

Reply:
Hmm, I guess lint config isn't set up to catch this in grist-core? Added await.